### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CaptVJ**
add description of game

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,5 +4,7 @@ our home page
 
 this is where people go to when they go to VmKRewritten DotCom
 
+VMKReWritten dOTcOM is a place to play VMK and have fun.
```
Comment (simcitynsun) [inline README.md:7]: really good copy but can you please update the domain name to match how it is written in the rest of the document ❤️

**Review by simcitynsun — CHANGES_REQUESTED** [review]

**Comment by simcitynsun** [comment]
also sorry but the readme doesnt show up on the website so this will only count for 5,000 credits. try editing index.html if you want the 25k credit bounty!!

File: README.md (hunk 1 above)
Comment (CaptVJ) [reply README.md:7]: ok i will try

**Comment by CaptVJ** [comment]
> also sorry but the readme doesnt show up on the website so this will only count for 5,000 credits. try editing index.html if you want the 25k credit bounty!!

ok i will try

**Review by simcitynsun — APPROVED** [review]
looks fab to me ❤️

**Pull request merged**